testsuite/src/test/java/org/ironjacamar/rars/ResourceAdapterFactory.java

```diff
@@ -557,18 +558,87 @@ else if (tsl == TransactionSupportLevel.LocalTransaction)
          org.ironjacamar.embedded.dsl.resourceadapters20.api.PoolType dashRaXmlPt = dashRaXmlCdt.getOrCreatePool()
                .minPoolSize(initialPoolSize).initialPoolSize(initialPoolSize).maxPoolSize(20).prefill(prefill)
                .flushStrategy(flushStrategy);
+
       }
       else
       {
          org.ironjacamar.embedded.dsl.resourceadapters20.api.XaPoolType dashRaXmlPt = dashRaXmlCdt.getOrCreateXaPool()
                .minPoolSize(initialPoolSize).initialPoolSize(initialPoolSize).maxPoolSize(20).prefill(prefill)
-               .flushStrategy(flushStrategy);;
+               .flushStrategy(flushStrategy);
       }
 
 
       return dashRaXml;
    }
 
+
+
+   /**
+    * Create the unified-security.rar
+    *
+    *
+    * @return The resource adapter archive
+    */
+   public static ResourceAdapterArchive createCapacityRar()
```

Comment:
I would just add extra parameters to the standard unified security configuration to make it explicit that it is unified.rar that is used